src/enums/effectOfOrderTypes.ts

```diff
@@ -1,4 +1,3 @@
 export enum EffectOfOrderTypes {
-  PLAN_OF_ARRANGEMENT = 'planOfArrangement',
-  ARRANGEMENT_EFFECT = 'arrangementEffect'
+  PLAN_OF_ARRANGEMENT = 'planOfArrangement'
```

Comment:
Does this enum still make sense, since it has only 1 value?

Reply:
This enum was there, before my previous changes. Other filings are using this too